sections/advanced/theming.js

```diff
@@ -117,6 +121,8 @@ const Theming = () => md`
   This is useful to circumvent a missing \`ThemeProvider\` or to override it.
 
   \`\`\`react
+  import styled, { ThemeProvider } from 'styled-components';
```

Comment:
This won't work as the `react` markdown snippets show a live editor, which doesn't support import statements right now.

I'd be happy to add a static `jsx` snippet with just the line of the import above the first example.

But I'm not sure as to whether this import isn't quite obvious :wink: Anyway, it'd be great to just add a single `jsx` snippet for this

Reply:
Hi, it might seem really obvious if you've used styled components before. But the ThemeProvider is a name that isn't unique to this project. Would difinitely save a few poor devs precious minutes.

I've updated the PR. How's that?